onto_tool/bundle_schema.yaml

```diff
@@ -0,0 +1,323 @@
+definitions:
+  non_empty_string:
+    type: string
+    minLength: 1
+  from_to:
+    type: object
+    properties:
+      from:
+        $ref: '#/definitions/non_empty_string'
+      to:
+        $ref: '#/definitions/non_empty_string'
+    required:
+      - from
+      - to
+    additionalProperties: false
+  onto_version:
+    type: object
+    properties:
+      iri:
+        $ref: '#/definitions/non_empty_string'
+      version:
+        $ref: '#/definitions/non_empty_string'
+    required:
+      - iri
+      - version
+    additionalProperties: false
+  include_list:
+    type: array
+    items:
+      $ref: '#/definitions/non_empty_string'
+    minLength: 1
+  bulk_file_operation:
+    properties:
+      source:
+        $ref: '#/definitions/non_empty_string'
+      target:
+        $ref: '#/definitions/non_empty_string'
+      rename:
+        $ref: '#/definitions/from_to'
+      replace:
+        $ref: '#/definitions/from_to'
+      includes:
+        $ref: '#/definitions/include_list'
+    required:
+      - source
+      - target
+  multi_query_spec:
+    oneOf:
+      - properties:
+          query:
+            $ref: '#/definitions/non_empty_string'
+        required:
+          - query
+      - properties:
+          queries:
+            type: object
+            properties:
+              source:
+                $ref: '#/definitions/non_empty_string'
+              includes:
+                $ref: '#/definitions/include_list'
+            required:
+              - source
+        required:
+          - queries
```

Comment:
This was added to support specifying either `query` or a list via `queries` for SELECT and ASK Verification.